crates/opencascade/src/primitives/shape.rs

```diff
@@ -542,7 +542,7 @@ impl Shape {
     }
 
     // TODO(bschwind) - Convert the return type to an iterator.
-    pub fn faces_along_ray(&self, ray_start: DVec3, ray_dir: DVec3) -> Vec<(Face, DVec3)> {
+    pub fn faces_along_ray(&self, ray_start: DVec3, ray_dir: DVec3) -> Vec<LineFaceHitPoint> {
```

Comment:
Let's go ahead and also rename this function to `faces_along_line` since that's technically what it's doing.